README.md

```diff
@@ -1,74 +1,80 @@
-# All your Express base are belong to us
+# Play
 
-[![Build Status](https://travis-ci.com/turingschool-examples/all-your-base.svg?branch=master)](https://travis-ci.com/turingschool-examples/all-your-base)
+## Introduction
 
-## Getting started
-To use this repo, you’ll need to `fork` the repo as your own. Once you have done that, you’ll need to run the following command below to get everything up and running. 
+Play is an Express API which allows existing users with a validated api key to hit four endpoints. Those endpoints include adding favorite songs, listing all favorited songs, finding a specific favorite and deleting a favorited song.
```

Comment:
```suggestion
Play is an Express API which allows users to hit four endpoints. Those endpoints include adding favorite songs, listing all favorited songs, finding a specific favorite and deleting a favorited song.
```

Reply:
@zacisaacson I made this edit because we don't require an API Key. As of right now, any visitor can send a request to the endpoints and POST, GET, or DELETE the data.